ld_script.ld

```diff
@@ -42,6 +42,7 @@ SECTIONS {
 
         /* COMMON starts at 0x30022A8 */
         INCLUDE "sym_common.ld"
+        src/*.o(COMMON);
```

Comment:
This won't include any `COMMON_DATA` symbols that users add to new files if they aren't explicitly included (i.e. we should wildcard `common_data` here too).

Reply:
I can imagine wildcarding COMMON not working previously because of the way that common symbols were previously included in the build (extracting them from the object files and then re-ordering them with alignment). However, since no variables in src/ use common variables anymore, we don't run into this problem.

The problem with wildcarding COMMON however occurs with common_data. `src/*.o(common_data);` in the linkerscript will cause the link process to run out of IWRAM, probably because the wildcarding isn't excluding files already in the build.